pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -0,0 +1,210 @@
+		*createdRes = append(*createdRes, &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": "multicluster.crd.antrea.io/v1alpha1",
+				"kind":       "ClusterSet",
+				"metadata": map[string]interface{}{
+					"name":      clusterSet.Name,
+					"namespace": clusterSet.Namespace,
+				},
+			},
+		})
+	}
+
+	return nil
+}
+
+func DeleteClusterClaim(cmd *cobra.Command, k8sClient client.Client, namespace string) error {
+	fmt.Fprintf(cmd.OutOrStdout(), "Deleting ClusterClaim \"%s\" in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimID, namespace)
+	if err := k8sClient.Delete(context.TODO(), newClusterClaim("", namespace, false)); err != nil && !apierrors.IsNotFound(err) {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete ClusterClaim \"%s\": %v\n", multiclusterv1alpha2.WellKnownClusterClaimID, err)
+		return err
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" deleted in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimID, namespace)
+	fmt.Fprintf(cmd.OutOrStdout(), "Deleting ClusterClaim \"%s\" in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimClusterSet, namespace)
+	if err := k8sClient.Delete(context.TODO(), newClusterClaim("", namespace, true)); err != nil && !apierrors.IsNotFound(err) {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete ClusterClaim \"%s\": %v\n", multiclusterv1alpha2.WellKnownClusterClaimClusterSet, err)
+		return err
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" deleted in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimClusterSet, namespace)
+	return nil
+}
+
+func DeleteClusterSet(cmd *cobra.Command, k8sClient client.Client, namespace string, clusterSet string) error {
+	fmt.Fprintf(cmd.OutOrStdout(), "Deleting ClusterSet \"%s\" in Namespace %s\n", clusterSet, namespace)
+	if err := k8sClient.Delete(context.TODO(), newClusterSet(clusterSet, namespace, "", "", "", "", "")); err != nil && !apierrors.IsNotFound(err) {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete ClusterSet \"%s\": %v\n", clusterSet, err)
+		return err
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "ClusterSet \"%s\" deleted in Namespace %s\n", clusterSet, namespace)
+	return nil
+}
+
+func newClusterClaim(name string, namespace string, clusterSet bool) *multiclusterv1alpha2.ClusterClaim {
+	clusterClaim := &multiclusterv1alpha2.ClusterClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+		},
+		Value: name,
+	}
+	if clusterSet {
+		clusterClaim.Name = multiclusterv1alpha2.WellKnownClusterClaimClusterSet
+	} else {
+		clusterClaim.Name = multiclusterv1alpha2.WellKnownClusterClaimID
+	}
+
+	return clusterClaim
+}
+
+func newClusterSet(name, namespace, leaderServer, secret, memberClusterID, leaderClusterID, leaderNamespace string) *multiclusterv1alpha1.ClusterSet {
+	if leaderServer != "" {
+		return &multiclusterv1alpha1.ClusterSet{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			},
```

Comment:
This part is shared, you may defined a variable first with this, and fill in the spec depends on `leaderServer`.